code/stable/gamephysics/Website/GamePhysics_SRS.html

```diff
@@ -1525,7 +1525,7 @@ <h3>Data Definitions</h3>
                       <th>Notes</th>
                       <td>
                         <p class="paragraph">
-                          The linear velocity <em><b>v</b><sub>B</sub></em> (m/s) of any point B in a rigid body (from <a href=#assumpOT>A: objectTy</a>) is the sum of the linear velocity <em><b>v</b><sub>O</sub></em> (m/s) of the rigid body at the origin (axis of rotation) and the resultant vector from the cross product of the rigid body's angular velocity <em>ω</em> (rad/s) and the displacement vector between the origin and point B, <em><b>r</b><sub>OB</sub></em> (m).
+                          The linear velocity <em><b>v</b><sub>B</sub></em> of any point B in a rigid body (from <a href=#assumpOT>A: objectTy</a>) is the sum of the linear velocity <em><b>v</b><sub>O</sub></em> of the rigid body at the origin (axis of rotation) and the resultant vector from the cross product of the rigid body's angular velocity <em>ω</em> and the displacement vector between the origin and point B <em><b>r</b><sub>OB</sub></em>.
```

Comment:
The Notes section here essentially restates the equation. Is this desired? How much redundancy is bad? @smiths

Reply:
Restating the equation isn't really necessary, but it doesn't bother me.  The words are sometimes helpful to readers, especially those that aren't strong with mathematical notation.  I don't think we always want the word version, but I also don't think we should delete all instances.  We can leave the Notes field for this DD as it is.